django_project/certification/models/course.py

```diff
@@ -0,0 +1,71 @@
+# coding=utf-8
+"""
+Course model definitions for certification apps
+"""
+
+from django.core.urlresolvers import reverse
+from django.db import models
+from course_convener import CourseConvener
+from certifying_organisation import CertifyingOrganisation
+from course_type import CourseType
+from training_center import TrainingCenter
+from django.contrib.auth.models import User
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Course(models.Model):
+    """Course model."""
+
+    name = models.CharField(
+        help_text="Course name.",
+        max_length=200,
+        null=True,
+        blank=True,
+    )
+
+    start_date = models.DateField(
+        _("Start date"),
```

Comment:
Its great that you marked this for translation - you should go back and check that all other text is marked too...